rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -187,15 +188,15 @@ public AccessIdRepresentationModel searchGroupByDn(final String name) {
     return accessId;
   }
 
-  public List<AccessIdRepresentationModel> searchGroupsofUsersIsMember(final String name)
+  public List<AccessIdRepresentationModel> searchGroupsAccessIdIsMemberOf(final String name)
       throws InvalidArgumentException {
-    LOGGER.debug("entry to searchGroupsofUsersIsMember(name = {}).", name);
+    LOGGER.debug("entry to searchGroupsAccessIdIsMemberOf(name = {}).", name);
     isInitOrFail();
     testMinSearchForLength(name);
 
     final AndFilter andFilter = new AndFilter();
-    andFilter.and(new WhitespaceWildcardsFilter(getGroupNameAttribute(), ""));
-    andFilter.and(new EqualsFilter(getGroupsOfUser(), name));
+    andFilter.and(new EqualsFilter(getGroupSearchFilterName(), getGroupSearchFilterValue()));
+    andFilter.and(new LikeFilter(getGroupsOfUser(), "*" + name + "*"));
```

Comment:
What's the reason for the like? Are we not searching strictly according to the accessId `name`?

Reply:
No. name could be a user, which we handle as uid olny. In the uniquemember field we do have the DN for users and groups.

Reply:
Then I don't like the name of the method. The methodname implies that the parameter is a accessId. Can we talk about this? That's faster than typing.